### ISSUE TITLE
Fails on a domain suffix of length 6 at dominatrex.ex:73

### ISSUE THREAD

**Issue by x-ji** [Zensavona/domainatrex#24]
The case statement at https://github.com/Zensavona/domainatrex/blob/9ee0b7d4a07924cfc1805957fea655710bc5b80f/lib/domainatrex.ex#L73 covers domain suffixes up to length 5, but apparently now the list fetched at https://raw.githubusercontent.com/publicsuffix/list/master/public_suffix_list.dat contains a domain suffix of length 6 (`.design`), and this causes `dominatrex` to fail to compile:

```
==> domainatrex
Compiling 1 file (.ex)

== Compilation error in file lib/domainatrex.ex ==
** (CaseClauseError) no case clause matching: 6
    lib/domainatrex.ex:73: anonymous fn/1 in :elixir_compiler_1.__MODULE__/1
    (elixir 1.15.5) lib/enum.ex:984: Enum."-each/2-lists^foreach/1-0-"/2
    lib/domainatrex.ex:71: (module)
```

**Comment by joerichsen** [comment]
A workaround for now is to add ```config :domainatrex, fetch_latest: false``` to your config.exs

**Comment by mrroot5** [comment]
It looks like the problem is related to this commit: https://github.com/publicsuffix/list/commit/80a8ca13fd7359bcf361f4f2b7e86ab0a508d5ea

**Comment by Zensavona** [comment]
This is fixed in `3.0.3`

**Comment by deepankar-j** [comment]
I'm seeing this issue on `v3.0.3`.  It looks like it's complaining about 7 now.

```
#32 64.89 == Compilation error in file lib/domainatrex.ex ==
#32 64.89 ** (CaseClauseError) no case clause matching: 7
#32 64.89     lib/domainatrex.ex:73: anonymous fn/1 in :elixir_compiler_3.__MODULE__/1
#32 64.89     (elixir 1.15.7) lib/enum.ex:984: Enum."-each/2-lists^foreach/1-0-"/2
#32 64.89     lib/domainatrex.ex:71: (module)
```
